tesseract/tesseract_visualization/src/trajectory_interpolator.cpp

```diff
@@ -61,7 +61,7 @@ TrajectoryInterpolator::TrajectoryInterpolator(tesseract_planning::CompositeInst
 
     // It is possible for sub composites to start back from zero, this accounts for it
     if (current_time < last_time)
-      last_time = current_time;
```

Comment:
This is correct?

Reply:
Yea, so if the current_time is less than the last_time then the time has started over. When it is set to the current_time the calculation of dt will be zero . The dt should be the current_time, because this only occurs on the first instruction when time restarts.